lib/middleware/profile.js

```diff
@@ -0,0 +1,18 @@
+module.exports = settings => {
+  const { Profile } = settings.models;
+  return (req, res, next) => {
+
+    if (req.user.profile) {
+      return next();
+    }
```

Comment:
This is in order to not totally screw @wheelsandcogs's integration tests which will set a profile manually.